exercises/transpose/TransposeExample.swift

```diff
@@ -0,0 +1,35 @@
+struct Transpose {
+    static func transpose(_ input: [String]) -> [String] {
+        let maxLineLength = input.map { $0.characters.count }.max()
+
+        guard let maxLength = maxLineLength else {
+            return []
+        }
+
+        var result = [String](repeatElement("", count: maxLength))
+
+        for i in 0..<maxLength {
+            for line in input {
+                if let start = line.index(line.startIndex, offsetBy: i, limitedBy: line.endIndex),
+                    let end = line.index(start, offsetBy: 1, limitedBy: line.endIndex) {
+                    let character = line[start..<end]
+                    result[i].append(character)
+                } else {
+                    result[i].append(" ")
+                }
+            }
+        }
+
+        return result.map { stripTrailingWhitespace($0) }
```

Comment:
good call. That would be an awesome rule to add to Swiftlint. You should suggest it.